scripts/jenkins/qat/plugin-deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Deploys current jenkins build test image 'intel-qat-plugin' in the cluster.
+
+set -o pipefail
+set -o xtrace
+set -o errexit
+
+REPO_ROOT=${WORKSPACE:-$(realpath $(dirname $0)/../../..)}
+REG=${REG:-intel/}
+TAG=${TAG:-devel}
+sed -i "s#intel/intel-qat-plugin:devel#${REG}intel-qat-plugin:${TAG}#g" ${REPO_ROOT}/deployments/qat_plugin/qat_plugin.yaml
```

Comment:
A global custom tag can be set using `scripts/set-version.sh`.  We may want to modify that to be able to override the REG path too.

Reply:
@mythi I just added a new commit that add the support to change tag, org or both: 

https://github.com/intel/intel-device-plugins-for-kubernetes/pull/231/commits/6bb0c6fa9065f4e89cd01b922f11bc5a2dd2e77a